### ISSUE TITLE
[bug] Typo in tabindex of FilterDropdown component

### ISSUE THREAD

**Issue by mvaldman** [unimelb/pattern-lib#1527]
tab-index="0", should be tabindex="0"
It's an issue that affects keyboard access of the drop-down control, because you can't tab to it
https://github.com/unimelb/pattern-lib/blob/dev/components/filters/components/filter-dropdown/FilterDropdown.vue

## Expected behaviour
 tabindex="0"

## Current behaviour
tab-index="0"

## Steps to reproduce
try and tab into the dropdown in story book: https://pattern-lib-unimelb.netlify.app/?selectedKind=filters%2Fcomponents%2FFilter%20Dropdown&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

or try in the new ATAR Calculator: https://study.unimelb.edu.au/change-of-preference/atar-calculator-2020

## Possible solution
Fix typo

**Comment by mvaldman** [comment]
@gepifanio I've noticed that tab-index appears in a number of components.

Is it possible that this is some kind of Vue.js feature? or is the typo more wide-spread than I thought?

![Screen Shot 2020-11-17 at 3 48 52 pm](https://user-images.githubusercontent.com/8953111/99348037-af195d00-28ec-11eb-946f-27c40127916c.png)
